components/AlbumEdit/AddPhoto.tsx

```
@@ -0,0 +1,94 @@
+	};
+
+	return (
+		<div className="modal">
+			<input
+				type="file"
+				accept="image/*"
+				onChange={handleFileChange}
+				multiple
+			/>
+			{photoData.map((photo, index) => (
+				<img src={photo} key={index} alt="選択写真プレビュー" width={100} />
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)